patches/spectre-console/0002-Remove-Mandelbrot-example-due-to-license.patch

```diff
@@ -0,0 +1,103 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Matt Thalman <mthalman@microsoft.com>
+Date: Wed, 3 Jan 2024 16:14:20 -0600
+Subject: [PATCH] Remove Mandelbrot example due to license
+
+Backport: https://github.com/spectreconsole/spectre.console/pull/1426
+---
+ examples/Console/Canvas/Mandelbrot.cs | 86 ---------------------------
+ 1 file changed, 86 deletions(-)
+ delete mode 100644 examples/Console/Canvas/Mandelbrot.cs
+
+diff --git a/examples/Console/Canvas/Mandelbrot.cs b/examples/Console/Canvas/Mandelbrot.cs
+deleted file mode 100644
+index da804a4..0000000
+--- a/examples/Console/Canvas/Mandelbrot.cs
++++ /dev/null
+@@ -1,86 +0,0 @@
+-/*
+-Ported from: https://rosettacode.org/wiki/Mandelbrot_set#C.23
+-Licensed under GNU Free Documentation License 1.2
```

Comment:
Will this actually address the license issue?  The license will still exist in this patch.

Reply:
But it wouldn't exist in the VMR.

Reply:
It would ([see here](https://github.com/dotnet/dotnet/tree/main/src/source-build-externals/patches)) - these patches get applied dynamically to the inner build of source-build-externals.  They are treated differently than the source-build patches.